### DISCUSSION TITLE
Disabled "Open connection in new tab" not respected with new connection through command line parameters when windows are closed

### DISCUSSION THREAD

**Discussion by gitterspec** [rustdesk/rustdesk#9751] [General]
### Bug Description

When run with command line parameters, new connections through command line open in a new tab of the existing window despite disabling this option in Settings. Only seems to happen from a starting point of all RustDesk windows being closed.

### How to Reproduce

Uncheck "Open connection in new tab" in Settings -> General
Close all RustDesk windows (confirm service is still running in system tray)
Run: C:\Program Files\RustDesk\rustdesk.exe --connect <IP:PORT> --password <PASS>
Run: C:\Program Files\RustDesk\rustdesk.exe --connect <IP2:PORT2> --password <PASS2>

### Expected Behavior

2nd new connection should open in a new window

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.3.1

### Screenshots

![image](https://github.com/user-attachments/assets/1de747a5-3579-4834-86bf-90cddcc9b1f8)

**Comment by rustdesk** [comment]
@fufesou low priority.

**Comment by gitterspec** [comment]
Hi, wondering if this should be an issue instead of a discussion? I've played around with it more and believe it's a reproducible bug under the settings described. It happens whenever new connections are launched through command line parameters when RustDesk windows are closed.

In terms of applicability, it affects setups that use hotkeys or shortcuts to launch multiple RustDesk connections.

**Comment by fufesou** [comment]
@gitterspec I've tested both `1.3.1` and the latest nightly build, but could not reproduce this issue.

```
"C:\Program Files\RustDesk\rustdesk.exe" --connect IP:PORT --password PASS1
"C:\Program Files\RustDesk\rustdesk.exe" --connect IP2:PORT2 --password PASS2
```

Please let me know if I'm missing something.

https://github.com/user-attachments/assets/c98204e0-f1f2-42fe-ad6c-86bdae58c41c

The video uses `.\rustdesk.exe`, but `"C:\Program Files\RustDesk\rustdesk.exe"` gets the same results.
